src/components/Card/CardSkeleton/CardSkeleton.js

```diff
@@ -3,39 +3,54 @@
  * @copyright IBM Security 2019
  */
 
+import classnames from 'classnames';
+import { PropTypes } from 'prop-types';
 import React from 'react';
 
 import { getComponentNamespace } from '../../../globals/namespace/index';
 
-import Card from '..';
+import Card from '../Card';
```

Comment:
Note that this was throwing a linting warning previously, because it was importing a default from the `index.js` instead of just directly from `Card/index.js`